package.json

```diff
@@ -50,7 +50,8 @@
   "dependencies": {
     "fs": "0.0.1-security",
     "mongoose": "^4.7.8",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "socket.io": "^1.7.2"
```

Comment:
`socket.io` should not be listed as a dependency but as a dev-dependency if we want to be able to run the tests. Thoughts? @jac1013